modules/priceFloors.js

```diff
@@ -751,7 +751,14 @@ export function addBidResponseHook(fn, adUnitCode, bid) {
     flooredBid.status = CONSTANTS.BID_STATUS.BID_REJECTED;
     // if floor not met update bid with 0 cpm so it is not included downstream and marked as no-bid
     flooredBid.cpm = 0;
-    logWarn(`${MODULE_NAME}: ${flooredBid.bidderCode}'s Bid Response for ${adUnitCode} was rejected due to floor not met`, bid);
+    const floorMsg = (() => {
+      try {
+        return ` (adjusted cpm: ${bid.floorData.cpmAfterAdjustments.toFixed(2)}, floor: ${floorInfo.matchingFloor.toFixed(2)})`
```

Comment:
I have a feeling the lack of precision is going to generate more bug reports when the rounded values are equal; how about toFixed(4)?

Reply:
how about keeping the full precision then? (because if 2 digits are not enough why would 4 be?)